pkg/kubectl/configmaps.go

```diff
@@ -2,37 +2,48 @@ package kubectl
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/pingcap/errors"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func UpsertConfigMap(data map[string]string, name string, namespace string) error {
 	ctx := context.Background()
-	_, cs, err := GetClientset()
+	_, cs, err := GetClientset() // Assuming GetClientset() returns the context and clientset
 	if err != nil {
-		panic(err)
+		return fmt.Errorf("failed to get clientset: %w", err)
```

Comment:
If we are going to return the error, we need to handle it in the calling function, which we currently do not, though panicing here is not bad per se, since there is not way to handle that particular failure

Reply:
I guess the right thing to do is check for the error and panic in the calling function

Reply:
Could you check if this is what you meant?